public/css/components.css

```diff
@@ -669,3 +673,45 @@ a.button {
     height:100%;
   }
 }
+/* Style inputs with type="text", select elements and textareas */
+input[type=text], select, textarea {
+  width: 100%; /* Full width */
+  padding: 12px; /* Some padding */
+  border: 1px solid #ccc; /* Gray border */
+  border-radius: 4px; /* Rounded borders */
+  box-sizing: border-box; /* Make sure that padding and width stays in place */
+  margin-top: 6px; /* Add a top margin */
+  margin-bottom: 16px; /* Bottom margin */
+  resize: vertical /* Allow the user to vertically resize the textarea (not horizontally) */
+}
+
+/* Style the submit button with a specific background color etc */
+input[type=submit] {
+  background-color: #04AA6D;
+  color: white;
+  padding: 12px 20px;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+}
+
+/* When moving the mouse over the submit button, add a darker green color */
+input[type=submit]:hover {
+  background-color: #45a049;
+}
+
+/* Add a background color and some padding around the form */
+.container {
+  border-radius: 2px;
+  background-color: #f2f2f2;
+  padding: 20px;
+  width: 300px;
+  margin: 10%;
+  padding: 10%;
+}
+.box {
+  border: 5px solid darkblue;
+  width: 300px;
+  margin: 10%;
+  padding: 10%;
+}
```

Comment:
FYI, modifying these components will change the entire website. It could potentially break every single page. 

We can go through the process of updating the components, however, I would create a separate PR for each component that we update. 

Alternatively, you could put all of this code in the `software-agency.css` file where it will safely only affect one page. You'll need to bump up the specificity so that it overrides the component CSS>